From f21f269302d8796fa794b6231cb116cc51eb92ab Mon Sep 17 00:00:00 2001
From: StefanBruens <stefan.bruens@rwth-aachen.de>
Date: Tue, 2 Jul 2019 20:37:02 +0200
Subject: [PATCH] Fix wxWidgets COMPONENTS specification, deprecated usage
 broken with CMake 3.14

Current FindWxWidgets.cmake allows to specify OPTIONAL components, unfortunately
this broke the (deprecated) use of wxWidgets_USE_LIBS.

Fixes: #779
---
 CMakeLists.txt | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 567f0fb7b48..427e0393a3e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -789,9 +789,7 @@ IF(BUILD_OSG_EXAMPLES AND NOT ANDROID)
 
     FIND_PACKAGE(FLTK)
     FIND_PACKAGE(FOX)
-
-    SET(wxWidgets_USE_LIBS base core gl net)
-    FIND_PACKAGE(wxWidgets)
+    FIND_PACKAGE(wxWidgets COMPONENTS base core gl net)
 
 ENDIF(BUILD_OSG_EXAMPLES AND NOT ANDROID)
 
